psql2mysql/__init__.py

```diff
@@ -275,6 +295,19 @@ def setupTypeDecorators(self, srcTable, targetTable):
                     targetTable.c[targetCol.name].type = decorator
 
 
+def positive_int(value):
+    try:
+        if not isinstance(value, int):
+            value = int(value)
+
+        if value < 0:
+            raise ValueError()
```

Comment:
Hmmm. This is almost a little dirty. Surely there is a more pythonic way ...

Reply:
Happy to go rework it :) this came up while I was debugging, I should have also written a test at the same time, so bad me :)